tests/integration/test_dataset.py

```diff
@@ -34,7 +34,7 @@ def test_dataset_missingness(dataset_name: str) -> None:
                 noise_type(dataset, config)
                 # Check missingness is synced with data
                 assert dataset.missingness.equals(dataset.is_missing(dataset.data))
-        else:
+        elif isinstance(noise_type, ColumnNoiseType):
```

Comment:
mypy started failing because it only knew that noise_type is a NoiseType, which doesn't have a __call__ method. However, we know that if we enter this block (if noise type is not a RowNoiseType), noise_type will be a ColumnNoiseType

Reply:
I'm open to letting mypy know in other ways though (like asserts) if people think that's better.

Reply:
nit: Maybe it's just me, but I don't like leaving `elif`s without a final `else`. Maybe just change this to another `if` instead?